Mesh_3/doc/Mesh_3/CGAL/Mesh_cell_criteria_3.h

```diff
@@ -37,15 +37,21 @@ typedef Tr::FT FT;
 /// @{
 
 /*!
-Returns an object to serve as default criteria for cells. The argument
-`radius_edge_bound` is the upper bound for the radius-edge ratio
-of the tetrahedra. The argument `radius_bound` is a uniform upper bound
-for the circumradii of the tetrahedra in the mesh. See
-section \ref introsecparam for further details.
-Note that if one parameter is set to 0, then its corresponding criteria is ignored.
+* Returns an object to serve as default criteria for cells.
+* @param radius_edge_bound is the upper bound for the radius-edge ratio
+*     of the tetrahedra.
+* @param radius_bound is a uniform upper bound
+*     for the circumradii of the tetrahedra in the mesh. See
+*     section \ref introsecparam for further details.
+* @param min_radius_bound is a uniform lower bound for the
+*     circumradii of the tetrahedra in the mesh.
+*     Only cells with a circumradius larger than that
+*     bound will be refined.
```

Comment:
@lrineau is this explanation not enough?

Reply:
ping @lrineau

Reply:
It sounds sufficient.